src/backend/looped/weapons/aimbot.cpp

```diff
@@ -96,15 +96,16 @@ namespace big
 				{
 					camera_target = aim_lock - CAM::GET_GAMEPLAY_CAM_COORD();
 				}
+				//We actually need this. For some unknow reasons it gets entity or something there.
 				if (aim_lock.x == 0.f && aim_lock.y == 0.f && aim_lock.z == 0.f)
 					return;
-				//PEMDAS looks good
 
-				float camera_heading = atan2f(camera_target.x, camera_target.y) * 180.0f / std::numbers::pi;
+				float RADPI = 180.0f / std::numbers::pi;
```

Comment:
I think the compiler would just optimize this out, but just to be sure, you should change this to a constexpr, so the compiler knows it can evaluate it at compile time.